src/components/AddWidget/AddWidget.tsx

```diff
@@ -98,6 +102,17 @@ const AddWidget = ({
             Add Guardian Ideal section
           </Button>
         )}
+        {flags && flags?.featuredShortcuts && (
```

Comment:
I think this would be enough to check the flag correct?

```suggestion
        {flags?.featuredShortcuts && (
```